client/extensions/woocommerce/store-sidebar/store-ground-control.js

```diff
@@ -27,7 +29,7 @@ const StoreGroundControl = ( { site, translate } ) => {
 			<div className="store-sidebar__ground-control-site">
 				<Site
 					compact
-					site={ site }
+					site={ isPlaceholder ? null : site }
```

Comment:
If `site === null` when it's a placeholder, you wouldn't have to have this ternary.